.github/workflows/deploy-aws-lambda.yaml

```diff
@@ -11,8 +11,11 @@ on:
     
 jobs:
   call-test-workflow:
-   # https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow
-   uses: ./.github/workflows/run-tests.yaml
+    uses: GNS-Science/nshm-github-actions/.github/workflows/python-run-tests.yml@main
+    with:
+      operating-systems: "['ubuntu-latest']"
```

Comment:
are we only testing ubuntu py10 because this is not a "user" package, only a deployed web api?

Reply:
I'm assuming so. That's what the deleted `run-tests.yaml` file did.